tasks/models.py

```diff
@@ -49,7 +49,7 @@ def net_duration(self):
         if duration := self.task_set.annotate(
             total_duration=models.F("duration") * models.F("repeats")
         ).aggregate(models.Sum("total_duration"))["total_duration__sum"]:
-            return duration
+            return duration * self.repeats
```

Comment:
Ok, now that I'm reading the help text again, I am actually thinking, the old version, which did not take session repeats into account is more accurate to the goal of this question. What do you think?